src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,26 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div className="card" key={movie.imdbId}>
+        <MovieCard {...movie} />
+      </div>
```

Comment:
Move this `div` inside `MovieCard` component.
```suggestion
      <MovieCard {...movie} key={movie.imdbId} />
```